### PR TITLE
Konstantinos /  browse our faq

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/pages/partners/deriv-prime/commercial-plan/index.tsx
[the remark this answers is not in the document]
Comment (aswathy-deriv) [reply src/features/pages/partners/deriv-prime/commercial-plan/index.tsx:1]: Its better we should check with designers team to use the same component for both as all the data and content is same.Not an ideal solution to create two separate components which has the same data

[the remark this answers is not in the document]
Comment (konstantinosG-deriv) [reply src/features/pages/partners/deriv-prime/commercial-plan/index.tsx:1]: Could do that yeah. I thought it wouldn't be good having 2 very similar components. But I guess it's cleaner to have separate components. Will do that. Thanks

File: src/features/pages/partners/deriv-prime/browse-our-faq/data.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
+import React from 'react'
+import CommercialPlan from '../commercial-plan'
+import { DerivPrimeFAQDataItem } from './types'
+
+export const generalFAQData: DerivPrimeFAQDataItem[] = [
```
Comment (aswathy-deriv) [inline src/features/pages/partners/deriv-prime/browse-our-faq/data.tsx:5]: can you change this variable name to `snake_case` something like `general_faq_data`

File: src/features/pages/partners/deriv-prime/browse-our-faq/data.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                    '_t_2. Verification: Satisfy all compliance and regulatory requirements including Know Your Business (KYB) requirements._t_',
+                    '_t_3. Margin account settlement: Fund your margin account to meet the minimum balance based on our scalable commercial plan._t_',
+                    '_t_4. Connectivity: Integrate with our platform using your preferred connectivity method._t_',
+                ],
+            },
+        ],
+    },
+]
+
+export const tocFAQDAta: DerivPrimeFAQDataItem[] = [
```
Comment (aswathy-deriv) [inline src/features/pages/partners/deriv-prime/browse-our-faq/data.tsx:79]: can you change this variable name to `snake_case` something like terms_cooperation_data

File: src/features/pages/partners/deriv-prime/browse-our-faq/deriv-prime-faq.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            justify="center"
+            align="center"
+            pb="20x"
+            md={{ pb: '40x' }}
+        >
+            <Typography.Paragraph
+                pb="12x"
+                align="center"
+                font_family="UBUNTU"
+                weight="normal"
```
Comment (aswathy-deriv) [inline src/features/pages/partners/deriv-prime/browse-our-faq/deriv-prime-faq.tsx:29]: not necessary to mention `weight='normal'`>>By default it takes normal

File: src/features/pages/partners/deriv-prime/browse-our-faq/deriv-prime-faq.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                                                        <Typography.Paragraph>
+                                                            <Localize
+                                                                translate_text={answerListItem}
+                                                            />
+                                                        </Typography.Paragraph>
+                                                    </li>
+                                                ))}
+                                        </ul>
+                                    ) : answerItem.type === 'custom_component' ? (
+                                        <div>{answerItem.component}</div>
```
Comment (aswathy-deriv) [inline src/features/pages/partners/deriv-prime/browse-our-faq/deriv-prime-faq.tsx:78]: `div` is needed here?

File: src/features/pages/partners/deriv-prime/browse-our-faq/data.tsx (hunk 1 above)
Comment (konstantinosG-deriv) [reply src/features/pages/partners/deriv-prime/browse-our-faq/data.tsx:5]: Sure, but is there a specific reason for this change?

File: src/features/pages/partners/deriv-prime/browse-our-faq/deriv-prime-faq.tsx (hunk 4 above)
Comment (konstantinosG-deriv) [reply src/features/pages/partners/deriv-prime/browse-our-faq/deriv-prime-faq.tsx:78]: Yes, for the alignment with the title but could use Flex.Box instead

File: src/features/pages/partners/deriv-prime/browse-our-faq/data.tsx (hunk 1 above)
Comment (aswathy-deriv) [reply src/features/pages/partners/deriv-prime/browse-our-faq/data.tsx]: yes all data varaiable should be in `snake_case`

**Review by hasan-deriv — APPROVED** [review]

**Review by mayuran-deriv — APPROVED** [review]

**Review by aswathy-deriv — APPROVED** [review]

**Pull request merged**